docs/limitations/nested-objects.asciidoc

```diff
@@ -0,0 +1,37 @@
+[[nested-objects]]
+== Limitation with nested objects
+
+Kibana is not able to perform aggregations across fields containing nested
+documents.
+
+[float]
+=== Workaround
+
+For simple aggregations (e.g. a `terms` aggregation), you
+may consider adding the `include_in_parent` setting to your Elasticsearch
+mapping. This creates a flattened version of a field in the nested document at
+the parent level.
+
+[source,js]
+----------------------------------
+{
+  "type1": {
+    "properties": {
+      "users": {
+        "type": "nested",
+        "include_in_parent": true,
+        "properties": {
+          "first": { "type": "keyword" },
+          "last": { "type": "keyword" }
+        }
+      }
+    }
+  }
+}
+----------------------------------
+
+[IMPORTANT]
+==============================================
+The `include_in_parent` workaround mentioned above is not a supported
+workaround. Please use it at your own risk.
+==============================================
```

Comment:
Ewww. Saying that a documented workaround is not supported is...not a great idea. It's sending a very mixed message. 

Can we flip it around and focus on why/when using it might be a bad idea? Something like: "In some cases, specifying `include_in_parent` to denormalize nested objects can lead to mapping explosion and have a serious performance impact. Use this workaround at your own risk and be sure to evaluate the performance implications for your use case. 

Editorial: In general, avoid "please" and other false politeness. It's fine to give users direct orders, you don't need to apologize for it.